light/client_benchmark_test.go

```diff
@@ -59,6 +59,9 @@ func (impl *providerBenchmarkImpl) LightBlock(ctx context.Context, height int64)
 func (impl *providerBenchmarkImpl) ReportEvidence(_ context.Context, _ types.Evidence) error {
 	panic("not implemented")
 }
+func (impl *providerBenchmarkImpl) String() string {
+	return ""
+}
```

Comment:
I'm not sure why we need this method. Can it get a more meaningful implementation?

```suggestion

func (impl *providerBenchmarkImpl) String() string {
	return ""
}
```

Reply:
Even just a placeholder value like `"fake-address-for-testing"` would be fine. Just something so the reader can see what it's for—it doesn't have to be "valid".